backend/models/tag.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (sequelize, Sequelize) => {
+  const Tag = sequelize.define('tag', {
+    id: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    title: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    created_at: {
+      type: Sequelize.DATE,
+      allowNull: false
+    },
+    updated_at: {
+      type: Sequelize.DATE,
+      allowNull: false
+    }
+  },
+  {
+    underscored: true
+  })
+
+  return Tag
+}
```

Comment:
Add newline at end of file

https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b